gradle/libs.versions.toml

```diff
@@ -0,0 +1,23 @@
+[versions]
+arrow = "1.1.2"
+kotlin = "1.6.21"
+kotest = "5.2.1"
+
+[libraries]
+kotest-runner-junit5 = { module = "io.kotest:kotest-runner-junit5", version.ref = "kotest" }
+kotest-assertions-core = { module = "io.kotest:kotest-assertions-core", version.ref = "kotest" }
+kotest-framework-engine = { module = "io.kotest:kotest-framework-engine", version.ref = "kotest" }
+kotest-framework-api = { module = "io.kotest:kotest-framework-api", version.ref = "kotest" }
+kotest-property = { module = "io.kotest:kotest-property", version.ref = "kotest" }
+
+kotlinx-coroutines-core = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version = "1.6.3" }
+
```

Comment:
the kotlin standard lib is missing here with version `1.6.21` - thanks to your PR I just realised that we have one in 1.7

Reply:
Kotlin gradle plugin automatically adds it to the sourceSets. Example dependencies dump:

```
jvmCompileClasspath - Compile classpath for compilation 'main' (target jvm (jvm)).
+--- io.arrow-kt:arrow-core:1.1.2
|    \--- io.arrow-kt:arrow-core-jvm:1.1.2
|         +--- io.arrow-kt:arrow-continuations:1.1.2
|         |    \--- io.arrow-kt:arrow-continuations-jvm:1.1.2
|         \--- io.arrow-kt:arrow-annotations:1.1.2
|              \--- io.arrow-kt:arrow-annotations-jvm:1.1.2
+--- io.kotest:kotest-assertions-core:5.2.1
|    \--- io.kotest:kotest-assertions-core-jvm:5.2.1
|         +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.10 -> 1.6.21
|         |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.6.21
|         |    |    \--- org.jetbrains:annotations:13.0
|         |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.6.21
|         |         \--- org.jetbrains.kotlin:kotlin-stdlib:1.6.21 (*)
|         \--- io.kotest:kotest-assertions-shared:5.2.1
|              \--- io.kotest:kotest-assertions-shared-jvm:5.2.1
|                   +--- io.kotest:kotest-assertions-api:5.2.1
|                   |    \--- io.kotest:kotest-assertions-api-jvm:5.2.1
|                   |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.10 -> 1.6.21 (*)
|                   \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.10 -> 1.6.21 (*)
\--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.21 (*)
```

I don't think it needs to be explicitly added anymore

Reply:
+1 😁